Library/Formula/spaceman-diff.rb

```diff
@@ -0,0 +1,44 @@
+class SpacemanDiff < Formula
+  desc "Diff images from the command line"
+  homepage "https://github.com/holman/spaceman-diff"
+  url "https://github.com/holman/spaceman-diff/archive/v1.0.tar.gz"
+  sha256 "934f38ae863abf15987100062268bcc28d37c39fe8c0a00a0813f07551944632"
+
+  head "https://github.com/holman/spaceman-diff.git"
+
+  depends_on "jp2a"
+  depends_on "imagemagick"
+  depends_on "roundup"
```

Comment:
I had included this to run [the tests](https://github.com/holman/spaceman-diff/blob/b04a8c46ba32a3f55aca5b175036a316049ec76e/tests#L6-L16), which you mentioned below I shouldn't use.